Standards/scs-XXXX-vN-key-manager-standard.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Key-Manager Standard
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+To encrypt user data like volumes or in the future also Images and ephemeral storage for VMs, the key is need to be known in the infrastructure.
+To provide the key to those operations without including the user every time a Key Manager within the infrastructure can be utilized to store the keys and apply authorization policies on requests.
```

Comment:
I needed to re-read this sentence 3 times before I understood the semantics. Please simplify